09/fish.py

```diff
@@ -0,0 +1,30 @@
+class Fish:
+    def __init__(self, type):
+        self.type = type
+    
+    def swim(self):
+        print('{} can swim in water.'.format(self.type))
```

Comment:
[Od Pythonu 3.6 lze používat takzvané *f-stringy*](https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498) namísto "tradiční" `str.format()` metody.

Čili toto by šlo napsat trochu kratší:
```python
print(f'{self.type} can swim in water.')
```

To tvoje není vyloženě špatně, ale říkám pro úplnost, že je tu ještě jiná možnost, která je často přehlednější a pro kód čitelnější. 😉